CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
-## v6.0.3
```

Comment:
Can we keep the 6.0.3 entry in the `CHANGELOG.md` please?

Reply:
oh, I messed it in the conflict resolution.